spec/lib/reporting/identity_verification_report_spec.rb

```diff
@@ -82,7 +83,7 @@
         ['Successfully Verified - With phone number', 1],
         ['Successfully Verified - With mailed code', 1],
         ['Successfully Verified - In Person', 1],
-        ['Successfully Verified - Passed fraud review', 1],
+        ['Successfully Verified - Passed fraud review', 2],
```

Comment:
Ok so now with this change, all the different successfully verified outcomes add up to 5, but total successfully verified is 4. I think people consuming this report will expect all the pieces to add up, is there any documentation we can update to indicate that there may be some overlap in these groups? (say for example, passed fraud review and also success with mailed in code)

Reply:
"Monthly Key Metrics Explainer" doc?

Reply:
I'll add in the doc